api4/plugin.go

```diff
@@ -127,16 +127,15 @@ func installPluginFromUrl(c *Context, w http.ResponseWriter, r *http.Request) {
 		force = true
 	}
 
-	fileBytes, readErr := ioutil.ReadAll(resp.Body)
-	if readErr != nil {
+	fileBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
 		c.Err = model.NewAppError("installPluginFromUrl", "api.plugin.install.reading_stream_failed.app_error", nil, err.Error(), http.StatusBadRequest)
 		return
 	}
 
-	manifest, unpackErr := c.App.InstallPlugin(bytes.NewReader(fileBytes), force)
-
-	if unpackErr != nil {
-		c.Err = unpackErr
+	manifest, appErr := c.App.InstallPlugin(bytes.NewReader(fileBytes), force)
```

Comment:
This is not part of the fix. But it's right next to the code I'm changing, hence I'm changing the code to follow the convention of `err` and `appErr`.